background/app/commands.js

```diff
@@ -4,15 +4,32 @@
 var ExtensionBackgroundDriver = ghs.libs.rpc.drivers.ExtensionBackgroundDriver;
```

Comment:
Since we already have one file named commands.js (which defines commands), it would be better to name this beast process-commands.js, for better readability.